plutarch-ledger-api/src/Plutarch/LedgerApi/AssocMap.hs

```diff
@@ -694,6 +695,67 @@ pdelete ::
   Term s (k :--> PMap 'Sorted k v :--> PMap 'Sorted k v)
 pdelete = rebuildAtKey # plam id
 
+{- | Build a function that zips two sorted 'PMap's together using a custom 'MergeHandler'.
+
+The provided 'MergeHandler' determines how to merge entries based on whether a
+key is present in the left map, the right map, or both.
+
+NOTE: This function itself cannot be hoisted with 'phoistAcyclic' because it
+depends on the supplied 'MergeHandler'. However, once you specialize it
+by providing a specific merge handler, the resulting function /should/ be
+hoisted if it will be reused, to avoid duplication.
```

Comment:
It's better to have notes be under their own heading. This would be written like so:

```
= Note

This function itself cannot be hoisted ....
```